interfaces/builtin/gpio_test.go

```diff
@@ -144,13 +151,31 @@ func (s *GpioInterfaceSuite) TestApparmorConnectedPlugIgnoresMissingSymlink(c *C
 	log, restore := logger.MockLogger()
 	defer restore()
 
+	builtin.MockEvalSymlinks(&s.BaseTest, func(path string) (string, error) {
+		c.Assert(path, Equals, "/sys/class/gpio/gpio100")
+		return "", os.ErrNotExist
+	})
+
 	spec := &apparmor.Specification{}
 	err := spec.AddConnectedPlug(s.iface, s.gadgetPlug, s.gadgetGpioSlot)
 	c.Assert(err, IsNil)
 	c.Assert(spec.Snippets(), HasLen, 0)
 	c.Assert(log.String(), testutil.Contains, "cannot export not existing gpio /sys/class/gpio/gpio100")
 }
 
+func (s *GpioInterfaceSuite) TestApparmorConnectedSlot(c *C) {
```

Comment:
I think you meant `TestApparmorConnectedPlug`?

Reply:
ah yes indeed, good catch